Spec/RealtimeClientChannel.swift

```diff
@@ -356,6 +356,40 @@ class RealtimeClientChannel: QuickSpec {
                     }
                 }
 
+                // RTL2f, TR4i
+                it("bit flag RESUMED was included") {
+                    let options = AblyTests.commonAppSetup()
+                    let client = ARTRealtime(options: options)
+                    defer { client.dispose(); client.close() }
+                    let channel = client.channels.get("test")
+
+                    waitUntil(timeout: testTimeout) { done in
+                        channel.attach() { error in
+                            expect(error).to(beNil())
+                            done()
+                        }
+                    }
+
+                    let attachedMessage = ARTProtocolMessage()
+                    attachedMessage.action = .Attached
+                    attachedMessage.channel = "test"
+                    attachedMessage.flags = 4 //Resumed
+
+                    waitUntil(timeout: testTimeout) { done in
+                        channel.once(.Update) { stateChange in
+                            guard let stateChange = stateChange else {
+                                fail("ChannelStageChange is nil"); done(); return
+                            }
+                            expect(stateChange.resumed).to(beTrue())
```

Comment:
In fact, might be worth checking that the resumed flag is always false on the first attach